### PR TITLE
Add a new achievement

### PR DESCRIPTION
**Pull request by optimumtact**
This achievement can be gained by slapping bubblegum with a tongue.

For science!

Sprite by Arcane

## Changelog
:cl: oranges
add: New achievements for science
/:cl:

### CODE REVIEW

File: code/modules/mob/living/simple_animal/hostile/megafauna/bubblegum.dm [hunk 1]
```diff
@@ -196,6 +196,21 @@ Difficulty: Hard
...
+  *
+  * Arguments:
+  * * obj/item/W the item hitting bubblegum
+  * * mob/user The user of the item
+  * * params, extra parameters
+  */
+/mob/living/simple_animal/hostile/megafauna/bubblegum/attackby(obj/item/W, mob/user, params)
+	. = ..()
+	if(istype(W, /obj/item/organ/tongue))
+		user.client.give_award(/datum/award/achievement/misc/frenching, user);
```
Comment (Rohesie) [inline code/modules/mob/living/simple_animal/hostile/megafauna/bubblegum.dm:212]:
```suggestion
		user.client?.give_award(/datum/award/achievement/misc/frenching, user);
```
You know how clients go, very volatile.

Comment (optimumtact) [reply code/modules/mob/living/simple_animal/hostile/megafauna/bubblegum.dm:212]: paranoia

**Comment by Twaticus** [comment]
what am i looking at

**Comment by optimumtact** [comment]
science

**Review by Twaticus — APPROVED** [review]
understandable have a great day

**Comment by optimumtact** [comment]
it's meant to be a tongue licking a gumball btw

File: code/modules/mob/living/simple_animal/hostile/megafauna/bubblegum.dm [hunk 2]
```diff
@@ -196,6 +196,21 @@ Difficulty: Hard
...
+  *
+  * Arguments:
+  * * obj/item/W the item hitting bubblegum
+  * * mob/user The user of the item
+  * * params, extra parameters
+  */
+/mob/living/simple_animal/hostile/megafauna/bubblegum/attackby(obj/item/W, mob/user, params)
+	. = ..()
+	if(istype(W, /obj/item/organ/tongue))
+		user.client?.give_award(/datum/award/achievement/misc/frenching, user);
```
Comment (Rohesie) [inline code/modules/mob/living/simple_animal/hostile/megafauna/bubblegum.dm:212]:
```suggestion
		user.client?.give_award(/datum/award/achievement/misc/frenching, user)
```
This is DM

Comment (optimumtact) [reply code/modules/mob/living/simple_animal/hostile/megafauna/bubblegum.dm:212]: cursed

**Pull request merged**